d8/d8batch/d8batch.install

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the D8: Batch demo module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function d8batch_install() {
+  \Drupal\menu_link_content\Entity\MenuLinkContent::create([
+    'title' => 'D8 Batch: Import feed sources',
+    'link' => ['uri' => 'internal:/import-feed-sources'],
+    'menu_name' => 'main',
+  ])->save();
+  \Drupal\menu_link_content\Entity\MenuLinkContent::load('dede')->delete();
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function d8batch_uninstall() {
+  $mids = \Drupal::entityQuery('menu_link_content')
```

Comment:
Yes there is a service so I've replaced this solution with it. Just a short notice: When you create a new link with the service it can not be edited on the UI (ex.: title), but with this solution the links could be modified on the UI.